client/lib/sites-list/test/test.js

```diff
@@ -1,104 +1,110 @@
-
-var SitesList = require( 'lib/sites-list/list' ),
-	Site = require( 'lib/site' ),
-	data = require( './data' ),
-	original = data.original,
-	updated = data.updated;
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import cloneDeep from 'lodash/cloneDeep';
+import forEach from 'lodash/forEach';
+
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'SitesList', () => {
+	let SitesList, Site, data;
+	useMockery();
+	useFakeDom();
+
+	before( () => {
+		Site = require( 'lib/site' );
+		SitesList = require( '../list' );
+		data = require( './fixtures/data' );
+	} );
 
-describe( 'SitesList', function() {
-	describe( 'initialization', function() {
-		it( 'should create Site objects', function() {
-			var sitesList = SitesList();
+	describe( 'initialization', () => {
+		let sitesList, originalData, initializedSites;
 
-			sitesList.initialize( original.slice() );
+		beforeEach( () => {
+			originalData = cloneDeep( data.original );
+			sitesList = SitesList();
+			sitesList.initialize( originalData );
+			initializedSites = sitesList.get();
+		} );
 
-			assert( sitesList.get().length === original.length );
+		it( 'should create the correct number of sites', () => {
+			assert.equal( initializedSites.length, originalData.length );
+		} );
 
-			sitesList.get().forEach( function( site ) {
-				assert( site instanceof Site );
+		it( 'should create Site objects', () => {
+			forEach( initializedSites, site => {
+				assert.instanceOf( site, Site )
 			} );
-
 		} );
 
-		it( 'should set attributes properly', function() {
-			var sitesList = SitesList(),
-				site, origSite;
-			sitesList.initialize( original.slice() );
-
-			site = sitesList.get()[0];
-			origSite = original[0];
-
-			for ( var prop in origSite ) {
-				assert( site[ prop ] ===  origSite[ prop ] );
+		it( 'should set attributes properly', () => {
+			let site = initializedSites[ 0 ];
+			let origSite = originalData[ 0 ];
 
-			}
+			forEach( origSite, ( value, prop ) => {
+				assert.deepEqual( value, site[ prop ] );
+			} );
 		} );
 
-		it( 'should add change handlers', function() {
-			var sitesList = SitesList();
-
-			sitesList.initialize( original.slice() );
-			assert( sitesList.get().length, original.length );
-			sitesList.get().forEach( function( site ) {
-				assert( site.listeners( 'change' ) );
+		it( 'should add change handlers', () => {
+			forEach( initializedSites, ( site ) => {
+				assert.isDefined( site.listeners( 'change' ) );
 			} );
-
 		} );
 	} );
 
-	describe( 'updating', function() {
-		it( 'updating should not create new Site instances', function() {
-			var sitesList = SitesList(),
-				originalList, updatedList;
+	describe( 'updating', () => {
+		let sitesList, originalData, updatedData, originalList;
 
-			sitesList.initialize( original.slice() );
+		beforeEach( () => {
+			originalData = cloneDeep( data.original );
```

Comment:
I guess we can move `originalData` and `updatedData` to parent before block to avoid code duplication and repeated initialization.